Description: Upstream changes introduced in version 2010-11-11
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 cvc4 (2010-11-11) unstable; urgency=low
 .
   * Initial release.
 .
 The person named in the Author field signed this changelog entry.
Author: Morgan Deters <mdeters@cs.nyu.edu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ cvc4-2010-11/config.reconfig
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Generated by configure, Thu Nov 11 22:49:37 EST 2010
+# This script is part of CVC4.
+
+cd "`dirname \"$0\"`"
+
+if test -d builds; then :; else echo 'No builds/ directory!' >&2; exit; fi
+
+current=(`grep '^CURRENT_BUILD' builds/current | sed 's,^CURRENT_BUILD *= *\([^/]*\)/\(.*\),\1 \2,'`)
+arch=${current[0]}
+build=${current[1]}
+
+echo "reconfiguring in builds/$arch/$build..."
+cd "builds/$arch/$build"
+echo ./config.status "$@"
+./config.status "$@"
